### DISCUSSION TITLE
ECS Service connect mentioned in documentation but cannot call attributes in CDK

### DISCUSSION THREAD

**Discussion by Munskin** [aws/aws-cdk#23334] [Q&A]
Hi,

Im trying to use ECS Service Connect and set it up using CDK. I've followed the examples provided here: https://docs.aws.amazon.com/cdk/api/v2/python/aws_cdk.aws_ecs/README.html#amazon-ecs-service-connect. Unfortunately I get the following error while using CDK commands:

```Traceback (most recent call last):
  File "//app.py", line 23, in <module>
    ecs = ECSCluster(
  File "//lib/python3.10/site-packages/jsii/_runtime.py", line 109, in __call__
    inst = super().__call__(*args, **kwargs)
  File "//services/ecs.py", line 263, in __init__
    ecs.ServiceConnectProps(
AttributeError: module 'aws_cdk.aws_ecs' has no attribute 'ServiceConnectProps'
```

I noticed that ECS Service Connect is pretty new, released a couple of weeks ago. Is this normal that there is documentation provided how to use it but it's not actually in the CDK version?

**Comment by daTobiGit** [comment]
which CDK version do you use in your project?
